Sources/MlemMiddleware/FeedLoaders/Generics/LoadingActor.swift

```diff
@@ -66,6 +66,16 @@ actor LoadingActor<Item: FeedLoadable> {
         print("[\(Item.self) LoadingActor] finished loading")
     }
     
+    @discardableResult
+    func filterItem(_ target: Item) -> Item? {
+        let filtered = filter.filter([target])
+        if filtered.isEmpty {
+            return nil
+        } else {
+            return filtered[0]
+        }
```

Comment:
*nit* this could be simplified

```suggestion
        let filtered = filter.filter([target])
        return filtered.first
```